docs/spatial.adoc

```diff
@@ -169,25 +169,3 @@ ORDER BY distance
 LIMIT 100
 ----
 
-== Combined Space and Time search
```

Comment:
where is the replacement for this? where the procedure calls are replaced with a function call?